cpp/src/arrow/compute/api_scalar.h

```diff
@@ -42,6 +42,55 @@ struct ArithmeticOptions : public FunctionOptions {
   bool check_overflow;
 };
 
+enum class RoundMode {
+  // floor (towards negative infinity)
+  DOWNWARD,
+  TOWARDS_NEG_INFINITY,
```

Comment:
If these are equivalent, shouldn't they have the same value?
```suggestion
  DOWNWARD,
  TOWARDS_NEG_INFINITY = DOWNWARD,
```